tests/plugins/redlinks/test_redlinks.py

```diff
@@ -0,0 +1,25 @@
+from tests.base import RequireRootArticleMixin, TestBase
+from wiki.core import markdown
+
+
+class RedlinksTests(RequireRootArticleMixin, TestBase):
+    def test_internal(self):
+        md = markdown.ArticleMarkdown(article=self.root_article)
+        md_text = md.convert("[Internal](.)")
+        self.assertIn("wiki-internal", md_text)
+        self.assertNotIn("wiki-external", md_text)
+        self.assertNotIn("wiki-broken", md_text)
```

Comment:
Could you maybe add something for the parent paths here, too? Would that make sense?

Reply:
I mean like `[Internal](../foo/bar)`

Reply:
Certainly - I've pushed a commit with more test coverage, now with tests that use a child article in addition to the root article. Please take another look.